src/test/java/com/softserveinc/dokazovi/aop/PostLoggerTest.java

```diff
@@ -187,18 +192,55 @@ void updatePostStatusNA() throws Throwable {
     }
 
     @Test
-    void deletePost() {
+    void deletePostBlankTitle() {
```

Comment:
This will get much simpler once you decouple audit logging from the aspect.

Reply:
Might be wrong regarding this specific one though, haven't used AOP for a long time, as we prefer different ways of expressing logic in our current project.